examples/wifi/wifi_aware/nan_console/main/nan_main.c

```diff
@@ -235,6 +235,11 @@ static int wifi_cmd_nan_disc(int argc, char **argv)
             nan_cfg.warm_up_sec = nan_args.warmup_time->ival[0];
         }
 
+        if(!g_nan_netif)
+        {
+            g_nan_netif = esp_netif_create_default_wifi_nan();    
+        }
+
         g_nan_netif = esp_netif_create_default_wifi_nan();
```

Comment:
You forgot to remove the unconditional initialization.

Reply:
I noticed the error after commit. Thanks for the observation!